test-harness/src/test/groovy/com/netflix/conductor/test/integration/JsonJQTransformSpec.groovy

```diff
@@ -148,4 +152,39 @@ class JsonJQTransformSpec extends AbstractSpecification {
             tasks[0].outputData.containsKey("result") && tasks[0].outputData.containsKey("resultList")
         }
     }
+
+    def "Test workflow with nested json jq transform task succeeds"() {
```

Comment:
s/nested/sequential/

Reply:
do you mean to rename from nested to sequential? My intention is to test the nested json object from jq_transform output, thus the prefix of "nested" naming

Reply:
Yes, thats the suggestion. I understand the intention here, but the naming of the spec and file indicate that the tasks are nested, consider renaming to ```json jq transform task with nested json object``` to explicitly call this out.